### PR TITLE
Update servlets and database to include additional information not coded last time

### PR DESCRIPTION
**Pull request by roonil-wazlib**
- add additional columns to cloud sql db for participant rank, rank yesterday, competition name etc
- update hard coded database entries to match db updates
- minor changes to gitignore and logging file
- update servlets to read in new information from database
- change all instances of 'ldap' to 'email' at backend
- some minor changes to variable types and naming

### CODE REVIEW

**Review by Phoebe-Khokgawe — APPROVED** [review]

File: project/src/main/java/com/google/sps/database/ConnectionPoolContextListener.java [hunk 1]
```diff
@@ -49,12 +49,12 @@
 
   private static String CREATE_COMPETITIONS_TABLE = 
     "CREATE TABLE IF NOT EXISTS competitions ( "
-    + "id SERIAL NOT NULL, start_date DATE NOT NULL, end_date DATE NOT NULL,"
+    + "id SERIAL NOT NULL, start_date DATE NOT NULL, end_date DATE NOT NULL, competition_name VARCHAR(255) NOT NULL, creator INT NOT NULL, creator_email VARCHAR(255) NOT NULL,"
```
Comment (leonnemets) [inline project/src/main/java/com/google/sps/database/ConnectionPoolContextListener.java:52]: {nit}: line getting long - split into 1 column per line

File: project/src/main/java/com/google/sps/servlets/CompetitionInfoServlet.java [hunk 2]
```diff
@@ -62,20 +62,26 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
    * @return -- all Competitions object that user is in.
```
Comment (leonnemets) [inline project/src/main/java/com/google/sps/servlets/CompetitionInfoServlet.java:62]: (Hard-coded for now) - not anymore?

File: project/src/main/java/com/google/sps/servlets/CompetitionInfoServlet.java [hunk 3]
```diff
@@ -85,16 +91,11 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
    * Construct a userCompetition object given data
    * @return -- userCompetition object
    */
-  private UserCompetition getUserCompetition(Connection conn, int userId, int competitionId, long start, long end) throws SQLException {
-    //not yet implemented
-    String COMP_NAME = "COMPETITION NAME";
-    String CREATOR = "CREATOR";
-    String CREATOR_IDAP = "CREATOR IDAP";
-
+  private UserCompetition getUserCompetition(Connection conn, int userId, int competitionId, long start, long end, String competitionName, int creatorId, String creatorEmail) throws SQLException {
```
Comment (leonnemets) [inline project/src/main/java/com/google/sps/servlets/CompetitionInfoServlet.java:94]: {nit } formatting

File: project/src/main/java/com/google/sps/servlets/InvestmentServlet.java [hunk 4]
```diff
@@ -55,10 +55,6 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       Gson gson = new Gson();
       response.setContentType("application/json");
       response.getWriter().println(gson.toJson(investments));
-
```
Comment (leonnemets) [inline project/src/main/java/com/google/sps/servlets/InvestmentServlet.java:58]: nice cleaning up

**Review by leonnemets — APPROVED** [review]

**Pull request merged**